inject.go

```diff
@@ -138,50 +144,58 @@ func (inj *injector) Apply(val interface{}) error {
 
 // Maps the concrete value of val to its dynamic type using reflect.TypeOf,
 // It returns the TypeMapper registered in.
-func (i *injector) Map(val interface{}) TypeMapper {
-	i.values[reflect.TypeOf(val)] = reflect.ValueOf(val)
-	return i
+func (inj *injector) Map(val interface{}) TypeMapper {
```

Comment:
Is there a strong reason for the s/i/inj/ changes? Non-functional changes (even if they're genuine improvements!) make this fork harder to push upstream in the future, which should always be the goal when working with a fork.